sp/metadata.php

```diff
@@ -28,6 +28,7 @@
  * @license    http://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
  */
 
+require_once('../../../config.php');
```

Comment:
I think the "convention" these days is to use __DIR__
like:
require(__DIR__. '/../../config.php');

https://docs.moodle.org/dev/What_happens_when_you_require_config.php